include/onnxruntime/core/optimizer/graph_transformer.h

```diff
@@ -85,8 +118,8 @@ each with different trade offs. At the moment, we define one that performs top-d
 */
 class RuleBasedGraphTransformer : public GraphTransformer {
```

Comment:
>RuleBasedGraphTransformer [](start = 6, length = 25)

Now might be a good time to split the rule based transformer classes out into a separate file (and possibly move them to the optimizers folder if possible).